packages/renderer/src/lib/feedback/feedbackForms/DevelopersFeedback.spec.ts

```diff
@@ -24,15 +24,19 @@ import { beforeAll, beforeEach, expect, test, vi } from 'vitest';
 
 import DevelopersFeedback from './DevelopersFeedback.svelte';
 
+const openExternalMock = vi.fn();
+const telemetryTrackMock = vi.fn();
+const sendFeedbackMock = vi.fn();
```

Comment:
Why declaring them on top? 

It should be avoid in favour of `vi.mocked(window.sendFeedback).mock....`